fendermint/vm/topdown/src/sync.rs

```diff
@@ -231,6 +240,29 @@ async fn sync_with_parent<T: ParentFinalityStateQuery + Send + Sync + 'static>(
     Ok(())
 }
 
+fn error_handling(
```

Comment:
`error_handling` is too general and it may lead to confusion, right? 
```suggestion
fn handle_parent_view_error(
```